### PR TITLE
Dev rework Comments & Notes

### PR DESCRIPTION
**Pull request by DocileThief**
This will be notes on Backend Functions and Files.

### CODE REVIEW

File: src/base/Window.js [hunk 1]
```diff
@@ -6,60 +6,84 @@ module.exports = class Window {
...
             if (this.name === 'main') {
+                //If function is main
                 if (config.minimizeToTray || windows.presets || windows.bars) {
                     windows[this.name].hide()
                     if (!windows.main?.isVisible() && !windows.presets && !windows.bars) quitHandler()
                 } else quitHandler()
+                //Options for closing/minimizing the tray
             } else {
-                if (this.name === 'ability') unregister()
+                //Couldnt find 'unregister'
```
Comment (txj-xyz) [inline src/base/Window.js:70]: `unregister()` is a `new global[]` that gets exposed from a module located here: https://github.com/txj-xyz/rs3-ability-tracker/blob/dev-rework/src/modules/UnregisterManager.js#L5

File: src/base/Window.js [hunk 2]
```diff
@@ -6,60 +6,84 @@ module.exports = class Window {
...
             delete windows[this.name]
             windows.tray.reload()
             if (!windows.main?.isVisible() && !windows.presets && !windows.bars) quitHandler()
             return void 0
+            //Remove th event listener, refresh windows tray, and if window is not visible, and not 'presets' and not 'bars', quit Handler.
         })
     }
 
     #emit(event) {
         for (const page in windows) !['properties', 'tray'].includes(page) ? windows[page].webContents.send(event, this.name) : void 0;
```
Comment (txj-xyz) [inline src/base/Window.js:86]: This line emits events for each window if we need it

Comment (DocileThief) [reply src/base/Window.js:86]: 👍

**Review by DocileThief — COMMENTED** [review]
From what I understand, this is called to create a new `Global[].`  I shouldn't have posted the commit, because I don't understand it all that well. What exactly does this file do in essence.

There is a comment in the file labeled `//Function to get frontend page paths.`. Is that referencing `module.exports = class Unregister extends Manager` or `static init() {
        return _ => { } }`. If I understand correctly, the module.exports is what the file returns when you do new UnRegister() : In so returning the static init function, that returns the thing I don't understand.

**Pull request closed without merging**